lib/Content/View/Builder/ContentCreateViewBuilder.php

```diff
@@ -0,0 +1,155 @@
+    ) {
+        $this->repository = $repository;
+        $this->viewConfigurator = $viewConfigurator;
+        $this->viewParametersInjector = $viewParametersInjector;
+        $this->defaultTemplate = $defaultTemplate;
+    }
+
+    public function matches($argument)
+    {
+        return 'ez_content_edit:createWithoutDraftAction' === $argument;
+    }
+
+    /**
+     * @param array $parameters
+     *
+     * @return ContentCreateView
+     *
+     * @throws InvalidArgumentException
+     * @throws UnauthorizedException
+     */
+    public function buildView(array $parameters)
+    {
+        // @todo improve default templates injection
+        $view = new ContentCreateView($this->defaultTemplate);
+
+        // @todo in 3.0 it should differentiate language and languageCode to allow injecting objects
+        if (isset($parameters['language'])) {
+            $language = $this->loadLanguage($parameters['language']);
+        } else {
+            throw new InvalidArgumentException(
+                'Language',
+                'No language could be loaded from parameters'
+            );
+        }
+
+        if (isset($parameters['parentLocationId'])) {
+            $location = $this->loadLocation($parameters['parentLocationId']);
+        } elseif (isset($parameters['parentLocation'])) {
+            $location = $parameters['parentLocation'];
+        } else {
+            $location = null;
+        }
+
+        if (isset($parameters['contentType'])) {
+            $contentType = $parameters['contentType'];
+        } else {
+            if (isset($parameters['contentTypeIdentifier'])) {
+                $contentTypeIdentifier = $parameters['contentTypeIdentifier'];
+            } else {
+                throw new InvalidArgumentException(
+                    'ContentType',
+                    'No content type could be loaded from parameters'
+                );
+            }
+
+            $contentType = $this->loadContentType($contentTypeIdentifier, [$language->languageCode]);
+        }
+
+        $view->setContentType($contentType);
+        $view->setLanguage($language);
+
+        if (isset($location)) {
+            $view->setLocation($location);
+        }
+
+        $view->addParameters([
+            'contentType' => $contentType,
+            'language' => $language,
+            'parentLocation' => $location,
+        ]);
+
+        $this->viewParametersInjector->injectViewParameters($view, $parameters);
+        $this->viewConfigurator->configure($view);
+
+        return $view;
+    }
+
+    /**
+     * @param string $languageCode
+     *
+     * @return Language
+     */
+    private function loadLanguage(string $languageCode): Language
+    {
+        return $this->repository->getContentLanguageService()->loadLanguage($languageCode);
+    }
+
+    /**
+     * @param $locationId
+     *
+     * @return \eZ\Publish\API\Repository\Values\Content\Location
+     */
+    private function loadLocation($locationId): Location
```

Comment:
For consistency's sake:
- please hint `int` and update PhpDoc `@param`
- please add `@throws` for `NotFoundException` and `UnauthorizedException`